src/main/java/core/CustomWebDriver.java

```diff
@@ -0,0 +1,69 @@
+package core;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Set;
+
+public class CustomWebDriver implements WebDriver {
+    protected WebDriver driver;
+
+    public CustomWebDriver(WebDriver driver) {
+        this.driver = driver;
```

Comment:
Just for better login lets add in constructor smth like
System.out.println("Custom web driver is used");

Reply:
Done